src/services/SPService.ts

```diff
@@ -559,14 +565,24 @@ export default class SPService implements ISPService {
            const isArray = Array.isArray(result[fieldName]);
            //multiselect lookups are arrays
            if (isArray) {
-            result[fieldName].forEach(element => {
-              lookups.push({ key: element.ID, name: element[lookupFieldName || 'Title'] });
+            result[fieldName].forEach(element => {                
+              let value = element[fieldName][lookupFieldName || 'Title'];
```

Comment:
This should be `element[lookupFieldName || 'Title']` because you've already iterate into the `fieldName` data.